packages/web-components/fast-components/src/listbox/scenarios/index.html

```diff
@@ -3,5 +3,6 @@
         <fast-option value="1">Option 1</fast-option>
         <fast-option value="2">Option 2</fast-option>
         <fast-option value="3">Option 3</fast-option>
```

Comment:
is this change related?

Reply:
Ah nm I see the change test

Reply:
yes. we can leave the change if that okay. I added option 4 or just remove it. I believe no harm was done on this.